canonical-web-and-design/ubuntu.com
# Add analytics events to subscription interactions.

## Done

- Add a util for sending analytics events.
- Send analytics events for each interaction in the UA table.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10442.demos.haus/advantage?test_backend=true) and log in.
- Click around the subscriptions list.
- Open your browser console and enter `dataLayer`.
- You should see events from the interactions you just performed.


## Issue / Card

Fixes: https://github.com/canonical-web-and-design/commercial-squad/issues/226.
